api/python/indigo/__init__.py

```diff
@@ -1275,8 +1275,8 @@ def stripSalt(self, inplace=False):
                            inorganic components from the molecule itself.
 
         Returns:
-            IndigoObject or None: new molecule without inorganic components
-                                  if inplace=False, None otherwise.
+            IndigoObject: new molecule without inorganic components
+                          if inplace=False, initial molecule otherwise.
```

Comment:
IMHO, something unclear is here:
`if inplace=False, initial molecule otherwise.`

"initial molecule" can be understood as initial molecule without any changes. I can only guess that it will be so if noting to remove...
Please give more detailed description.

Reply:
Yes, it's indeed wrong. Fixed it too!